pkg/artifactory/resource_artifactory_local_repository.go

```diff
@@ -85,6 +85,11 @@ var legacyLocalSchema = mergeSchema(map[string]*schema.Schema{
 		Optional: true,
 		Computed: true,
 	},
+	"priority_resolution": {
```

Comment:
Ok, this we don't want to support. Usage of the old blob type repo was bad. We now have a local repo resource for every single repo type supported by RT. The legacy repo is intentionally left to rot and die.

Reply:
sounds good, I'll get rid of it

Reply:
done